### DISCUSSION TITLE
Commands not running after javascript:install:[bundler]

### DISCUSSION THREAD

**Discussion by bjornlinder** [rails/jsbundling-rails#201] [Q&A]
I've tried intalling a couple different bundlers now for a new app, essentially getting the same error message in both cases. 'bun' or 'esbuild' is not recognized a valid command. Bun looks to be the simplest installation so we can use that example, where in the [installation script](https://github.com/rails/jsbundling-rails/blob/main/lib/install/bun/install.rb) is the bun command actually added? Does it attempt to add a symlink to path or is a task added directly to the rails app? Current theory is that my current terminal setup isn't well suited for symlinks (gitbash+zsh+windows).

`./bin/rails javascript:install:bun` installation goes fine.

`the_best git:(master) ✗ bin/dev
20:07:23 web.1  | started with pid 4824
20:07:23 js.1   | started with pid 13740
20:07:23 css.1  | started with pid 16112
20:07:23 js.1   | unknown command: bun run build --watch
20:07:23 js.1   | yarn run v1.22.22
20:07:23 css.1  | yarn run v1.22.22
20:07:23 js.1   | $ bun bun.config.js --watch
20:07:24 css.1  | error Command "build:css" not found.
20:07:24 css.1  | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
20:07:24 js.1   | 'bun' is not recognized as an internal or external command,
20:07:24 js.1   | operable program or batch file.
20:07:24 js.1   | error Command failed with exit code 1.
20:07:24 js.1   | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
20:07:24 js.1   | exited with code 1
`

**Comment by dhh** [comment]
There are not of lot of people working on straight Windows compatibility issues. I would strongly recommend using WSL2. From the error messages, this looks like you don't have bun installed in the execution context.

**Reply by bjornlinder** [reply]
Thanks for your response. Yes, in some ways making things harder for myself working directly on windows. I did figure out another major conflict in my setup, in that node/yarn install don't work directly on google drive. As of now, bun is installed and rails server works, but bin/dev still doesn't recognize bun. Good enough for moving forward.
